src/components/Anchor/index.js

```diff
@@ -21,13 +21,14 @@ const prefixByType = {
 const Anchor = ({
   url,
   children,
+  className = '',
```

Comment:
I don't like adding className props to components as it opens the door for misuse; instead of `className='ma1'` being passed, I would add a `div` or `span` wrapper with those styles around the reusable component; that way you keep instances / layout styles separate from components and have predictable component rendering.